drivers/sdhc/sdhc_cdns_ll.c

```diff
@@ -720,8 +721,9 @@ static int sdhc_cdns_send_cmd(struct sdmmc_cmd *cmd, struct sdhc_data *data)
 
 	timeout = CARD_REG_TIME_DELAY_US;
 	if (!WAIT_FOR(((((sys_read32(cdns_params.reg_base + SDHC_CDNS_SRS12)) &
-		CDNS_SRS03_CMD_DONE) == 1) | (((sys_read32(cdns_params.reg_base +
-		SDHC_CDNS_SRS12)) & ERROR_INT) == ERROR_INT)), timeout, k_busy_wait(1))) {
+			CDNS_SRS12_CC) == CDNS_SRS12_CC) | (((sys_read32(cdns_params.reg_base +
+			SDHC_CDNS_SRS12)) & CDNS_SRS12_EINT) == CDNS_SRS12_EINT)),
+			timeout, k_busy_wait(1))) {
```

Comment:
can we use k_msleep instead of k_busy_wait?

Reply:
done